packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.tsx

```diff
@@ -22,6 +22,8 @@ export interface CheckboxProps
   id: string;
   /** Aria-label of the checkbox. */
   'aria-label'?: string;
+  /** Description text of the checkbox. */
+  description?: string;
```

Comment:
is `string` limiting in any way? I tried putting an HTML tag in `description` and it didn't work. I think it's fair for that section to contain links or maybe elements like `<strong>`, `<span>`, etc.

Reply:
Yes, you're right - `string` is limit. I can change that to `React.Node` which allows `<strong>`, `<i>` and etc.